test/functional/rpc_blockchain.py

```diff
@@ -451,6 +452,7 @@ def _test_getnetworkhashps(self):
         )
         # This should be 2 hashes every 10 minutes or 1/300
         assert abs(hashes_per_second * 300 - 1) < 0.0001
+        assert abs(hashes_per_second_since_diff_adjustment * 300 - 1) < 0.006
 
     def _test_stopatheight(self):
```

Comment:
nit in commit and maybe PR title.
[Use the imperative mood in the subject line](https://cbea.ms/git-commit/#imperative)
`test: add coverage to rpc_blockchain.py `
see https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#committing-patches

Reply:
updated the title and cherry-picked your commit 17e505a